scripts/update-issues.ts

```diff
@@ -293,17 +296,17 @@ async function update() {
       continue;
     }
 
-    if (data.status.baseline) {
+    const title = data.name;
+    const body = issueBody(id, data);
+    const issue = openIssues.get(id);
+    
+    if (data.status.baseline && !issue) {
```

Comment:
The two above reasons for skipping (vendor positions and a feature becoming discouraged) could also happen for a feature with an open issue, which would then become orphaned by the script.

We'll need to take different action in each of these cases (some combination of comment/close/lock) but we can punt on that part for now. But I think that if we have a helper function that just returns the skip reason, we can the do something similar to what you have here, and always update the issue if there is one, but not create a new one.

Reply:
Thank you, let's go with just TODOs!